tasks/render-resource-pipeline-templates.yml

```diff
@@ -0,0 +1,13 @@
+platform: linux
+image_resource:
+  type: registry-image
+  source: {repository: bitnami/jsonnet}
+inputs:
+- name: pipelines
+outputs:
+- name: rendered_pipelines
+params:
+  RESOURCES:
+run:
+  path: pipelines/tasks/scripts/render-resource-pipeline-templates
+  user: root
```

Comment:
why do we need to explicitly tell that it's root? is it because `bitnami/jsonnet` has a different user set by default that concourse doesn't detect?

Reply:
Something wacky about that image, yeah: https://hub.docker.com/r/bitnami/jsonnet/dockerfile the `USER 1001` doesn't have permission to read from the task input or write to the task output. Might be worth using a different image, I just liked the sales pitch.